Pod/SQL/GDGEntity+SQL.m

```diff
@@ -297,10 +297,16 @@ - (BOOL)save:(NSError **)error
 		saved = [db.table update:values error:error];
 	}
 	else
-		saved = [db.table insert:values error:error];
+	{
+		NSString *insertId = [db.table insert:values];
+
+		NSNumberFormatter *f = [[NSNumberFormatter alloc] init];
```

Comment:
I suggest you to rename `f` with `formatter` or `numberFormatter`.

Reply:
Outdated :P